### ISSUE TITLE
Recreating Results

### ISSUE THREAD

**Issue by crest123-max** [FactoDeepLearning/VerticalAttentionOCR#4]
Hello, I'm still relatively new to working with larger python projects so this may have an obvious answer, but what are the instructions for recreating the results? I have recreated the environment in Google Colab, formatted the dataset, placed the downloaded model in the checkpoint, edited the transfer_learning parameter to map to the downloaded model in main_pg_va.py, and commented out the model.train() line in main_pg_va.py. I ran the script and it didn't give me any errors, but the CER I'm getting is 1. This leads to me thinking that I'm doing something wrong. Can you help point me in the right direction? Thank you so much for your help, I really appreciate it!

**Comment by FactoDeepLearning** [comment]
Hello, thanks for reporting me this issue. I guess you did it right, I've update the model file (a crucial line was missing). It should be ok now !

**Comment by crest123-max** [comment]
That fixed the problem and I'm now getting the expected results. Thanks so much and I appreciate the timely response!

**Comment by jjsr** [comment]
Hi, I am training the model from scratch . I have enabled the 'use_ddp' option in training parameters. Rest is default as given.  I am on 1672 epoch CTC loss is 5.669 CER is 0.7126 and WER is 1.0072(it's kind of frozen) diff_length is 0.422. My last best epoch is 1018 . I think I may be doing something wrong. May anyone please help. Thanking in advance. Should I turn off DDP and run it again. Please Please help
